pkg/cliplugins/workload/plugin/syncer.yaml

```diff
@@ -13,6 +13,15 @@ metadata:
   name: {{.ServiceAccount}}
   namespace: {{.Namespace}}
 ---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{.ServiceAccount}}-token
+  namespace: {{.Namespace}}
+  annotations:
+    kubernetes.io/service-account.name: {{.ServiceAccount}}
```

Comment:
I still do not understand this whatsoever - this is making a token on the pcluster, right? We specifically don't want it to be an SA token, or annotated, right? Since if it is, the downstream controller deletes it?